Automattic/jetpack
# Site Logo: change enqueued CSS file depending on lang. & setti…

#### Changes proposed in this Pull Request:

* When someone uses the `SCRIPT_DEBUG` constant, or has their site set to a RTL Language, they should get the right stylesheet enqueued.

#### Testing instructions:

* Add Site Logo support to your theme:
```php
$args = array(
	'header-text' => array(
		'site-title',
		'site-description',
	),
	'size' => 'medium',
);
add_theme_support( 'site-logo', $args );
```
* Go to Appearance Customize, and check what files are enqueued on the page.
* You should see `css/site-logo-control.min.css` being enqueued.
* Go to Settings > General and switch your site language to Hebrew.
* You should see `css/site-logo-control-rtl.min.css` being enqueued.
* Add `define( 'SCRIPT_DEBUG', true );` to `wp-config.php`
* You should see `css/site-logo-control-rtl.css` being enqueued.

#### Proposed changelog entry for your changes:

* Site Logo: ensure that the right stylesheet is loaded depending on your site's language.
